tests/testthat/outputs/vars-pull-input-checking.txt

```diff
@@ -0,0 +1,51 @@
+> vars_pull(letters, letters)
+Error: Must select with column names or positions.
+x `var` has the wrong size 26.
+i This index must be size 1.
+
+> vars_pull(letters, aa)
+Error in eval_tidy(enquo(var), set_names(seq_along(vars), vars)): object 'aa' not found
+
+> vars_pull(letters, 0)
+Error: Must select with column names or positions.
+x `var` can't be zero.
+i This index must be a positive integer.
+
+> vars_pull(letters, 100)
+Error: Must select existing columns.
+x Can't subset position 100.
+i There are only 26 elements.
+
+> vars_pull(letters, -100)
+Error: Must select existing columns.
+x Can't subset position 100.
+i There are only 26 elements.
+
+> vars_pull(letters, -Inf)
+Error: Must select with column names or positions.
+x Lossy cast from `var` <double> to <integer>.
+
+> vars_pull(letters, TRUE)
+Error: Must select with column names or positions.
+x `var` has the wrong type `logical`.
+i This index must be a position or a name.
+
+> vars_pull(letters, NA)
+Error: Must select with column names or positions.
+x `var` can't be `NA`.
+i This index can't be missing.
```

Comment:
This seems a bit uninformative, but I think it's fine to keep if it's hard to eliminate.